rpc_jobs/rpc_openstack.yml

```diff
@@ -604,10 +604,9 @@
     image:
       - bionic_mnaio_no_artifacts:
           SLAVE_TYPE: "nodepool-ubuntu-bionic-om-io2"
-      - xenial_mnaio_no_artifacts:
```

Comment:
These two changes are completely unrelated. Can you break them into separate commits?

Reply:
@johnduarte Updated.